dev/ci/ci-wrapper.sh

```diff
@@ -14,7 +15,27 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "${DIR}/../.." || exit 1
 
 export TIMED=1
-bash "${DIR}/${CI_SCRIPT}" 2>&1 | tee "$CI_NAME.log"
+
+# NB: in CI TERM is unset in the environment
+# when TERM is unset, bash sets it to "dumb" as a bash variable (not exported?)
+if { [ -t 1 ] && ! [ "$TERM" = dumb ]; } || [ "$CI" ]
+then color_wanted=1
+else color_wanted=
+fi
+
+if [ "$color_wanted" ] && command -v script > /dev/null; then
+  # prevent piping from disabling auto colors / enable auto colors in CI
+    if [ "$CI" ]; then
+      export TERM=xterm-color
+      export GIT_PAGER=cat
```

Comment:
Rather than forcing the output of all the git commands to go through `cat`, it might be better to just disable the pager.

```suggestion
      export GIT_PAGER=""
```

And here is a variant that does not depend on environment variables:

```suggestion
      git config --global core.pager ""
```

Reply:
We don't want to hit the global config when this runs on a developer's machine. Developers may want to run with the CI variable set for debugging weird errors.

Reply:
Good point. Then just the empty pager.